api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -40,6 +40,10 @@ message TCPAccessLogEntry {
 
   // Properties of the TCP connection.
   ConnectionProperties connection_properties = 2;
+
+  // If this log entry is final log entry that flushed after the connection closed or
+  // intermediate log entry that flushed periodically during the connection.
+  bool intermediate_log_entry = 3;
```

Comment:
When this is true, how do an ALS determine whether two TCPAccessLogEntry is from the same connection or not?

Reply:
then, we need a unique id for this. A uuid would be better than the connection id.

Reply:
```
  // All metadata encountered during request processing, including endpoint
  // selection.
  //
  // This can be used to associate IDs attached to the various configurations
  // used to process this request with the access log entry. For example, a
  // route created from a higher level forwarding rule with some ID can place
  // that ID in this field and cross reference later. It can also be used to
  // determine if a canary endpoint was used or not.
  config.core.v3.Metadata metadata = 17;
```

How about inject a uuid to the metadata of TCPAccessLogEntry. Seems like this field is designed for this scene.
If we need determine whether multiple intermediate log entry if from same connection or not, the the related id should be provided in the metadata.

Reply:
Good suggestion, I forgot a connection ID(any ID to represent the TCP connection) is crucial .

Reply:
Sounds, good, worth a comment in the documentation.